tutorials/CIFAR_Tutorial.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import torch.utils.data.dataloader as dataloader
+from resnet import ResNet18
+from torch.autograd import Variable
+from torch.utils.data import Dataset
+from torchvision import transforms
+from torchvision.datasets import CIFAR10
+
+from metal import EndModel
+from metal.utils import convert_labels
+
+SEED = 1
+
+# CUDA?
+cuda = torch.cuda.is_available()
+
+# For reproducibility
+torch.manual_seed(SEED)
+
+if cuda:
+    torch.cuda.manual_seed(SEED)
+
+
+parser = argparse.ArgumentParser(description="PyTorch Training")
```

Comment:
This is a tutorial so being slightly more verbose with explanatory comments is good- add one here saying e.g. 'setting up command line argument parser'